packages/testlab/package.json

```diff
@@ -23,7 +23,7 @@
     "@types/express": "^4.11.1",
     "@types/fs-extra": "^5.0.1",
     "@types/shot": "^3.4.0",
-    "@types/sinon": "^4.3.0",
+    "@types/sinon": "^5.0.2",
```

Comment:
This is breaking - https://travis-ci.org/strongloop/loopback-next/jobs/435495036

Reply:
Interesting, why locally all tests are ok?:

Reply:
Hmm, maybe I did not correctly reinstall dependencies? IDK.